.github/workflows/create-cli-deps-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: "Create CLI Deps PR"
+
+on:
+  workflow_dispatch:
+    inputs:
+      npmVersion:
+        description: "NPM Version"
+        required: true
+        default: '"6.x.x" or "latest"'
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.NPM_ROBOT_TOKEN }}
+      NPM_VERSION: ${{ github.event.inputs.npmVersion }}
+    steps:
+      - name: Checkout npm/node
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: master
+          repository: "npm/node"
+          token: ${{ secrets.NPM_ROBOT_TOKEN }}
```

Comment:
@gimli01 is this the right secret/token reference? I don't see this on our side. Potentially this should be `NPM_ROBOT_USER_PAT`?

Reply:
@darcyclarke this workflow actually uses `npm/node` and then clones the cli into the workspace. this secret lives in `npm/node`: https://github.com/npm/node/settings/secrets/actions